manifest_staging/charts/csi-secrets-store-provider-azure/templates/_helpers.tpl

```diff
@@ -24,19 +24,41 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Common labels for helm resources
+*/}}
+{{- define "sscdpa.common.labels" -}}
+app.kubernetes.io/instance: "{{ .Release.Name }}"
+app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+{{- end -}}
+
 {{/*
 Standard labels for helm resources
 */}}
 {{- define "sscdpa.labels" -}}
 labels:
-  app.kubernetes.io/instance: "{{ .Release.Name }}"
-  app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+{{ include "sscdpa.common.labels" . | indent 2 }}
   app.kubernetes.io/name: "{{ template "sscdpa.name" . }}"
-  app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
   app: {{ template "sscdpa.name" . }}
-  helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
 {{- end -}}
 
 {{- define "sscdpa.psp.fullname" -}}
 {{- printf "%s-psp" (include "sscdpa.fullname" .) -}}
 {{- end }}
+
+{{/*
+Arc specifics templates
+*/}}
+
+{{- define "sscdpa.arc.fullname" -}}
+{{- printf "%s-arc" (include "sscdpa.fullname" .) -}}
```

Comment:
Why do we need a arc fullname? Could we just reuse `sscdpa.fullname` instead and when extension is installed it can configure the `-arc` suffix with values?